fastapi_template/template/{{cookiecutter.project_name}}/deploy/docker-compose.yml

```diff
@@ -56,9 +56,13 @@ services:
       {{cookiecutter.project_name | upper}}_DB_PORT: {{cookiecutter.db_info.port}}
       {{cookiecutter.project_name | upper}}_DB_USER: {{cookiecutter.project_name}}
       {{cookiecutter.project_name | upper}}_DB_PASS: {{cookiecutter.project_name}}
+      {%- if cookiecutter.db_info.name == "mongodb" %}
+      {{cookiecutter.project_name | upper}}_DB_BASE: admin
```

Comment:
Can we name the base with the name of the project as for other DBs?

Reply:
That would require populating another config file called `mongo-init.js` and mounting it into the container. See https://stackoverflow.com/a/54064268 for more info.

Do you think it's better to include this in this PR, or I open another one to implement this? Either one works for me.

Thanks!

Reply:
I guess separate PR is fine. Let's not think about it now. Because as I think it's quite complex to set up custom database name.